### PR TITLE
CVE and error logging fixes

### PR DESCRIPTION
**Pull request by runcom**
@mtrmac PTAL, see individual commits messages

### CODE REVIEW

**Comment by runcom** [comment]
skopeo PR running here https://github.com/projectatomic/skopeo/pull/451

File: signature/policy_config.go [hunk 1]
```diff
@@ -70,7 +70,11 @@ func NewPolicyFromFile(fileName string) (*Policy, error) {
 	if err != nil {
 		return nil, err
 	}
-	return NewPolicyFromBytes(contents)
+	policy, err := NewPolicyFromBytes(contents)
+	if err != nil {
+		return nil, errors.Wrapf(err, "invalid policy data in %q", fileName)
```
Comment (mtrmac) [inline signature/policy_config.go]: `s/data//`?

**Comment by mtrmac** [comment]
:+1:

… but \*grumble\* about adding an unrelated cosmetic bug fix into a security fix PR.

**Comment by runcom** [comment]
Well, they're different commits haha, lemme rebase this, drop "data" and merge

**Comment by mtrmac** [comment]
> Well, they're different commits haha

True enough, that’s why I gave the :+1: at all.

**Comment by TomSweeneyRedHat** [comment]
Code LGTM, but it look like you need to rebase.

**Comment by runcom** [comment]
LGTM

**Pull request merged**